FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift

```diff
@@ -43,7 +42,7 @@ class ModelDownloadTask: NSObject {
   /// User defaults to which local model info should ultimately be written.
   private let defaults: UserDefaults
   /// Keeps track of download associated with this model download task.
-  private(set) var downloadStatus: ModelDownloadStatus
+  private(set) var downloadStatus: ModelDownloadStatus = .complete
```

Comment:
Why `.complete` is the default state?

Reply:
Ah I reduced the possible statuses to be either `complete` or `inProgress`. Earlier, I had a `notStarted` as well but it seemed a bit redundant. I can bring it back for clarity though.

Reply:
It looks a bit confusing when initial status is `complete` to me. But I'm mostly fine with either way (not a blocker for the PR for me).

Reply:
Ok makes sense, I brought back `.notStarted` to clear this up.